alonet/torch2trt/base_exporter.py

```diff
@@ -188,37 +204,20 @@ def adapt_graph(self, graph):
         graph: onnx_graphsurgeon.Graph
         """
         return graph
-    
+
     def _adapt_graph(self, graph, **kwargs):
         """Modify ONNX graph to ensure compability between ONNX and TensorRT
 
         Returns
         -------
         graph: onnx_graphsurgeon.Graph
         """
-        clip_nodes = get_nodes_by_op("Clip", graph)
-        def handle_op_Clip(node: gs.Node):
-            max_constant = np.array(np.finfo(np.float32).max, dtype=np.float32)
-            if "value" in node.inputs[1].i().inputs[0].attrs:
-                min_constant = node.inputs[1].i().inputs[0].attrs["value"].values.astype(np.float32)
-                if len(node.inputs[2].inputs) > 0:
-                    max_constant = node.inputs[2].i().inputs[0].attrs["value"].values.astype(np.float32)
-            elif "to" in node.inputs[1].i().inputs[0].attrs:
-                min_constant = np.array(np.finfo(np.float32).min, dtype=np.float32)
-            else:
-                raise Exception("Error")
```

Comment:
the function is important for certain models for some tensorrt version. A better fix is to replace the exception with a `return`